fastchat/serve/gradio_block_arena_named.py

```diff
@@ -175,19 +178,27 @@ def add_text(
                 no_change_btn,
             ]
             * 6
+            + [True]
```

Comment:
maybe use a variable say `show_vote_button`? I see below we have a few more `[True]` `[False]`